redaxo/src/core/assets/standard.js

```diff
@@ -130,6 +134,42 @@ if (opener != null)
     var winObjCounter = -1;
 }
 
+function rex_retain_popup_event_handlers(eventName) {
+    if (!opener || !opener.rex || !window.name) {
+        return;
+    }
+
+    var events = opener.rex.popupEvents || {};
+
+    if (events[window.name] && events[window.name][eventName]) {
```

Comment:
Sagen wir mal so: Zumindest in den Fällen, wo die Funktion bisher genutzt wird, ja!
Linkmap-Popups haben immer den Namen "linkmappopup", und es kann nur ein offenes Linkmap-Popup geben.
Der Medienpool kann verschachtelt geöffnet werden, dort hat aber jedes Popup einen eigenen Namen "rexmediapopupX" (X fortlaufende Nummer).